build/README.md

```diff
@@ -299,17 +306,37 @@ and makes cross-platform support for the build system much easier.
 To push your own images into the cluster, take a look at Minikube's
 [Pushing Images](https://minikube.sigs.k8s.io/docs/handbook/pushing/) guide.
 
-Running end-to-end tests on Minikube can be done via the `make minikube-test-e2e` target, but this can often overwhelm
-a local minikube cluster, so use at your own risk. Take a look at
-[Running Individual End-to-End Tests](#running-individual-end-to-end-tests) to run individual tests on a case by case
-basis.
+Running end-to-end tests on Minikube can be done via the `make minikube-test-e2e` target, but this can often overwhelm a local minikube cluster, so use at your own risk. Take a look at [Running Individual End-to-End Tests](#running-individual-end-to-end-tests) to run individual tests on a case by case basis.
 
 If you are getting issues connecting to `GameServers` running on minikube, check the
 [Agones minikube](https://agones.dev/site/docs/installation/creating-cluster/minikube/) documentation. You may need to
 change the driver version through the `MINIKUBE_DRIVER` variable. See the
 [local-includes](#set-local-make-targets-and-variables-with-local-includes) on how to change this permanently on
 your development machine.
 
+#### Running e2e tests with LoadBalancer support (MetalLB)
+
+Minikube does not natively support LoadBalancer services, which are required for some Agones e2e tests. To enable LoadBalancer support locally, you must install and configure [MetalLB](https://metallb.universe.tf/) in your Minikube cluster:
+
+1. Run the following Makefile targets to install and configure MetalLB:
+   ```sh
+   make minikube-metallb-helm-install
+   make minikube-metallb-configure
+   ```
```

Comment:
Do we need this section now it's installed by default?

Reply:
Not needed anymore 👌🏼